airbyte-webapp/src/pages/OnboardingPage/components/SkipOnboardingButton.tsx

```diff
@@ -1,31 +0,0 @@
-import React from "react";
```

Comment:
:information_source: This file was a dead file, that I came across beause it also uses the `finishOnboarding` method, so I removed it as part of this PR.